schema/spacewalk/common/tables/rhnServer.sql

```diff
@@ -60,6 +60,7 @@ CREATE TABLE rhnServer
                             CONSTRAINT rhn_server_mtsched_id_fk
                                 REFERENCES suseMaintenanceSchedule (id)
                                 ON DELETE SET NULL,
+    auto_sync           BOOLEAN default true NOT NULL,
```

Comment:
This should go to `rhnchannel.sql` I guess, not `rhnserver.sql`

Reply:
Thank you Thomas, I think I was blind on this one :|